cpu/cc2538/Makefile.include

```diff
@@ -1,3 +1,8 @@
 export CPU_ARCH := cortex-m3
-
+	ROM_LEN ?= 0x80000
+  	RAM_LEN ?= 0x8000
+	ROM_START_ADDR ?= 0x00200000
+	RAM_START_ADDR ?= 0x20000000
+	LINKER_SCRIPT ?= cc2538sf53.ld
+USEMODULE += periph_common
```

Comment:
Why is this `USEMODULE` needed ?

Reply:
I don't think it's needed. It was in the original RIOT base which I was working with. I've noticed it's no longer in the master base, so I'll remove it now